todolist/parser.go

```diff
@@ -109,14 +110,21 @@ func (p *Parser) parseArbitraryDate(_date string, pivot time.Time) string {
 
 func (p *Parser) parseArbitraryDateWithYear(_date string, year int) time.Time {
 	res := strings.Join([]string{_date, strconv.Itoa(year)}, " ")
-	if date, err := time.Parse("Jan 2 2006", res); err == nil {
+	var date time.Time
+	var err1 error
+	var err2 error
```

Comment:
I wanted to preserve the two different errors, so do the declaration explicitly.   Please refer to my replies below for more details.